packages/database/src/sql/postgres.ts

```diff
@@ -63,8 +63,14 @@ export class Postgresql extends BaseSql {
    * @param {string} database
    */
   async clearDatabase(database: string) {
-    await exec(`dropdb ${database}`).catch(() => true)
-    await exec(`createdb ${database}`).catch(() => true)
+    await exec(`dropdb ${database}`).catch((err) => {
+      console.log('Clear DB: Drop DB')
+      console.log(err)
+    })
+    await exec(`createdb ${database}`).catch((err) => {
+      console.log('Clear DB: Create DB')
+      console.log(err)
+    })
```

Comment:
to help with future debugging. this was essential in finding out what caused the tests to fail on github actions